cmd/geth/main.go

```diff
@@ -39,7 +39,7 @@ import (
 	"github.com/ethereum/go-ethereum/log"
 	"github.com/ethereum/go-ethereum/metrics"
 	"github.com/ethereum/go-ethereum/node"
-	"gopkg.in/urfave/cli.v1"
+	cli "gopkg.in/urfave/cli.v1"
```

Comment:
Aliasing `cli` is not required

Reply:
thanks, I agree,  should I raise a new PR for this or we commit this and raise a quick PR to revert cli alias change if required?

alias, of course, does not have any impact and has nothing to do with go 11 changes.  it was auto-added.

Reply:
You can commit the change and push the change from the original branch and it will update the PR

Reply:
many thanks , updated now.